models/org.go

```diff
@@ -256,6 +256,10 @@ func CountOrganizations() int64 {
 
 // DeleteOrganization completely and permanently deletes everything of organization.
 func DeleteOrganization(org *User) (err error) {
+	if !org.IsOrganization() {
+		return fmt.Errorf("%s is an user not a organization", org.Name)
```

Comment:
```suggestion
		return fmt.Errorf("%s is an user not an organization", org.Name)
```

`a`, `an` infinitive

Reply:
```suggestion
		return fmt.Errorf("%s is a user not an organization", org.Name)
```